tests/Crypto/KmsMaterialsProviderTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Aws\Test\Crypto;
+
+use Aws\Crypto\KmsMaterialsProvider;
+use Aws\Crypto\MetadataEnvelope;
+use Aws\Kms\KmsClient;
+use Aws\Result;
+use Aws\Test\UsesServiceTrait;
+
+/**
+ * @covers Aws\Crypto\KmsMaterialsProvider
+ */
+class KmsMaterialsProviderTest extends \PHPUnit_Framework_TestCase
+{
+    use UsesServiceTrait;
+
+    public function testProviderBasics()
+    {
+        /** @var KmsClient $client */
+        $client = $this->getTestClient('Kms', []);
+        $keyId = '11111111-2222-3333-4444-555555555555';
+
+        $provider = new KmsMaterialsProvider($client, $keyId);
+
+        $this->assertEquals(
+            ['kms_cmk_id' => $keyId],
```

Comment:
From context, I'm guessing that `CEK` stands for "content encryption key" and `CMK` stands for "content master key"; why not just use `EncryptionKey` and `MasterKey` as the parameter names?